api/rs/slint/mcu.md

```diff
@@ -1,25 +1,25 @@
 # Slint on Microcontrollers (MCU)
 
-This document explains how to use Slint to develop a UI on a MCU in a bare metal environment.
+The following scetions explain how to use Slint to develop a UI on a MCU in a bare metal environment.
 
-## Install Toolchain / Hardware Abstraction Layer (HAL)
+## Prerequisites
 
-Writing an application in Rust that runs on a MCU requires the following software components:
+Writing an application in Rust that runs on a MCU requires several prerequisites:
 
-* You need to install a Rust toolchain to cross-compile to the target architecture.
-* You need to locate and select the correct HAL crates and drivers, and depend on them in your `Cargo.toml`.
-* You need to install tools for flashing and debugging your code on the device.
+* Install a Rust toolchain to cross-compile to the target architecture.
+* Locate and select the correct Hardware Abstraction Layer (HAL) crates and drivers, and depend on them in your `Cargo.toml`.
+* Install tools for flashing and debugging your code on the device.
 
-Covering these is out of scope for this document, but we recommend reading the [Rust Embedded Book](https://docs.rust-embedded.org/book/),
+We recommend reading the [Rust Embedded Book](https://docs.rust-embedded.org/book/),
 as well as the curated list of [Awesome Embedded Rust](https://github.com/rust-embedded/awesome-embedded-rust) for a wide range of different
 crates, tools and training materials. These resources should guide you through the initial setup and often come with "hello world" examples
 to get started with your device.
 
-In order to set a global allocator, required by Slint, you will need a **nightly** version of Rust. This is due to the fact that the support for using a custom global allocator in a bare metal
-environment with `#![no_std]` has not been stabilized yet (see [#51540](https://github.com/rust-lang/rust/issues/51540) or
+Slint requires a global memory allocator. That is available the nightly version of Rust, because the support for using a custom global
```

Comment:
```suggestion
Slint requires a global memory allocator. That is only possible with the nightly version of Rust, because the support for using a custom global
```

Reply:
I used "That is currently only available in the nightly version" to emphasise that this will change in the future (hopefully :-). Is that ok?

Reply:
That's Ok,  although i liked "possible" more :-)

Reply:
Ahh, you've convinced me. After all, they might decide to implement this using different API if they stabilise it, and then it's about the possibility of doing it, not about the availability of the concrete API that's in nightly right now. I'll change it :)